### PR TITLE
Normalize travis with head of quality tools

### PR DESCRIPTION
**Pull request by nbessi**

### CODE REVIEW

**Comment by nbessi** [comment]
Before merging:
- [x] wait  https://github.com/OCA/maintainer-quality-tools/pull/57 to be fixed and merged
- [x] change the branch of dreispt to master

File: .travis.yml [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
-services:
-  - postgresql
+virtualenv:
+  system_site_packages: true
 
 install:
-    - $HOME/maintainer-quality-tools/travis/travis_install_nightly 7.0
-    - pip install coveralls flake8
-
+ - git clone https://github.com/dreispt/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools -b add-include-env
```
Comment (pedrobaeza) [inline .travis.yml]: You should wait until this PR would be merger to make changes, although as a "test of the tests" can be valid.

Comment (nbessi) [inline .travis.yml]: @pedrobaeza c.f. first comment :smile:

**Comment by bwrsandman** [comment]
:+1:

**Comment by nbessi** [comment]
It should be ready to be merged, just waiting travis to finish.
Flake 8 will be activated in a separated PR as line length is now 80 and I do not have the time to fix it now

**Comment by pedrobaeza** [comment]
I think you shouldn't deactivate flake8 although it's not yet corrected. Any future PR until the fix will not check it.

**Comment by nbessi** [comment]
Well it is mostly a matter of separation of concern.
My previous work to fix pep8 was big. 
Now that line len is 80 I want a separated patch to fix it.

**Comment by nbessi** [comment]
> flake8 . --max-line-length=80 --exclude=**unported**,**init**.py,scenarios| grep 501 | wc -l
> 435

So if there is any volunteer to help it would be nice

**Comment by sbidoul** [comment]
Do you know why account_partner_required is in the EXCLUDE section? Last time I checked, tests were ok for this one.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by nbessi** [comment]
@sbidoul Test are fine the module just needs to be tested separately has it has uncompatible constraint(s) with other addons

**Comment by sbidoul** [comment]
Hm. account_partner_required should play nice with other modules because the default policy is optional.

Actually, the problem is either
- in the account_partner_required tests which calls write with an id instead of a list of ids
- or in account_constraints which does not do the usual dance: if isinstance(ids, (int, long)): ids = [ids]

I never remember. Do you know which one is right?

**Comment by nbessi** [comment]
PEP8 WIP:
https://github.com/nbessi/account-financial-tools/tree/fix_pep_8_80

**Comment by yvaucher** [comment]
:+1:

**Comment by sbidoul** [comment]
:+1: the EXCLUDE/INCLUDE and pep8 can be sorted out later

**Pull request merged**